### PR TITLE
foot

### PR DESCRIPTION
**Pull request by LordGrey1**
Done the bottom part and changed stars to png

### CODE REVIEW

**Comment by LordGrey1** [comment]
new star

File: index.html [hunk 1]
```diff
@@ -32,8 +32,8 @@ <h1 class="maintitle">World Finest Tennessee Whiskey</h1>
     <img src="star.jpg" class="star" />
     <div class="titlefont">Story - Taste - Character</div>
   </div>
-  <div class="Stepmom">
+  <div class="foot">
     <h1><b>THE STORY</b></h1>
```
Comment (naimmustafa) [inline index.html]: please use css instead of h1 and b tags

File: syle.css [hunk 2]
```diff
@@ -37,4 +37,12 @@ body {
...
     font-size: 100px;
     font-weight: bolder;
+}
+.foot{
+    background-color: blueviolet;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-weight: bolder;
+    font-size: 20px;
```
Comment (naimmustafa) [inline syle.css]: nice!

Comment (naimmustafa) [inline syle.css]:
```suggestion
    font-weight: bolder;
    font-size: 20px;
```
these are making h1 and b tags useless, you should move them to `THE STORY`s own class

**Review by naimmustafa — APPROVED** [review]

**Pull request merged**